roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -40,6 +40,12 @@ openshift_logging_es_pvc_prefix: "{{ openshift_hosted_logging_elasticsearch_pvc_
 # config the es plugin to write kibana index based on the index mode
 openshift_logging_elasticsearch_kibana_index_mode: 'unique'
 
+openshift_logging_elasticsearch_oauth_proxy_image_prefix: "openshift/oauth-proxy"
+openshift_logging_elasticsearch_oauth_proxy_image_version: "v1.0.0"
+openshift_logging_elasticsearch_oauth_proxy_cpu_limit: "100m"
+openshift_logging_elasticsearch_oauth_proxy_memory_limit: "32Mi"
```

Comment:
I do not have any context but this seems a bit small?

Reply:
I am open to suggestions. I was trying to minimize cpu and mem requirements